pyoomph/c_output.py

```diff
@@ -690,10 +745,53 @@ def _define_simple_type(self, the_type: Type) -> None:
             else:
                 raise NotImplementedError(name)
 
+    def _define_functype(self, functype: FunctionType) -> None:
+        # TODO: equals method
```

Comment:
There is already an equals method, along with a comment about how it's not ideal. But maybe functions should sometimes be hashable:

```suggestion
        # TODO: hash method?
```